diff -Naur orig.merecat/configure.ac merecat/configure.ac
--- orig.merecat/configure.ac	2019-04-22 03:53:33.036317659 -0700
+++ merecat/configure.ac	2019-04-22 16:09:27.805177859 -0700
@@ -181,9 +181,12 @@
 AC_SUBST(SBINDIR)
 WWWDIR=`eval echo $WEBDIR`
 AC_SUBST(WWWDIR)
+SYSCONFDIR=`eval echo $sysconfdir`
+AC_SUBST(SYSCONFDIR)
 
 AC_CONFIG_FILES([Makefile
                  merecat.service
+				 merecats.service
                  src/Makefile
                  man/Makefile
                  scripts/Makefile
diff -Naur orig.merecat/Makefile.am merecat/Makefile.am
--- orig.merecat/Makefile.am	2019-04-22 03:53:33.032984421 -0700
+++ merecat/Makefile.am	2019-04-22 15:40:46.424373708 -0700
@@ -4,11 +4,11 @@
 DISTCHECK_CONFIGURE_FLAGS = --with-systemd=$$dc_install_base/$(systemd)
 
 if HAVE_CONFUSE
-dist_sysconf_DATA         = merecat.conf
+dist_sysconf_DATA         = merecat.conf merecats.conf
 endif
 
 if HAVE_SYSTEMD
-systemd_DATA              = merecat.service
+systemd_DATA              = merecat.service merecats.service
 endif
 
 ## Generate .deb package
diff -Naur orig.merecat/merecat.conf merecat/merecat.conf
--- orig.merecat/merecat.conf	2019-04-22 03:53:33.036317659 -0700
+++ merecat/merecat.conf	2019-04-22 03:54:08.318637102 -0700
@@ -70,7 +70,7 @@
 
 ## When chrooting, alt. document root inside chroot
 ## => /var/www/htdocs
-#data-directory = /htdocs
+#data-directory = /srv/http
 
 ## Skip dotfiles in dirlistings
 #list-dotfiles = false
diff -Naur orig.merecat/merecats.conf merecat/merecats.conf
--- orig.merecat/merecats.conf	1969-12-31 16:00:00.000000000 -0800
+++ merecat/merecats.conf	2019-04-22 15:40:23.898354591 -0700
@@ -0,0 +1,95 @@
+## /etc/merecat.conf                                     -*-conf-unix-*-
+## This is a sample configuration file for Merecat httpd
+## For more help and more settings, see merecat(8)
+##
+
+## what interface to bind to?
+## (default is binding to any interface)
+#hostname=www.example.org
+
+## Port to listen to, overrides command line argument
+## Defaults to 80, or 443 when enabling HTTPS
+port = 443
+
+## Enable basic HTTPS support
+ssl = true
+
+## The HTTPS certificate (public) and key (private)
+## are required when enabling HTTPS support
+#certfile = certs/cert.pem
+#keyfile = private/key.pem
+
+## Unpriviliged user to run as, usually nobody or www-data
+#username = nobody
+
+## CGI:
+##	?	match a single char
+##	*	matches any string excluding "/"
+##	**	matches any string including "/"
+##	separate multiple patterns with "|"
+##Example: "**.sh|**.cgi"
+# If no CGI pattern is set, CGIs are disabled files are
+# read as-is.   See the next section for more information.
+#cgi-pattern = "**.cgi|/cgi-bin/*"
+
+## Interpreter
+## In addition or instead of a CGI pattern, match file pattern
+## and pass it to a specified program.  Especially useful for PHP files
+## without having to modify them to run as an executable.  If 
+## interpreter is not set, interpreter-pattern is ignored.
+#interpreter-pattern = "**.php"
+#interpreter = "/usr/bin/php-cgi"
+
+# Max number of simultaneous CGI programs allowed.
+#cgi-limit = 1
+
+## Global .htpasswd (true) or local per-directory (false)
+#global-passwd = false
+
+## Chrooting is a security measure which means isolating the webserver's
+## access to files only available from a the given directory.  To access
+## files outside the chroot the administrator can either copy or bind
+## mount files and directories into the chroot.
+#chroot = false
+
+## Only useful if not chrooting
+#check-symlinks = false
+
+## Alt. charset=iso-8859-1
+#charset = UTF-8
+
+## Deflate (gzip) compression level: -1 .. 9
+## -1: Default (zlib's reasonable default, currently 6)
+##  0: Disabled
+##  1: Best speed
+##  9: Best compression
+#compression-level = -1
+
+## Webserver document root, or chroot
+#directory = /var/www
+
+## When chrooting, alt. document root inside chroot
+## => /var/www/htdocs
+#data-directory = /srv/http
+
+## Skip dotfiles in dirlistings
+#list-dotfiles = false
+
+## Virtual hosting
+## /var/www/cgi-bin/          <-- Shared CGI
+## /var/www/git.example.com   <-- git.example.com
+## /var/www/ftp.example.com   <-- ftp.example.com
+#virtual-host = false
+
+## Control the caching, in seconds, by setting the following header for
+## all transactions.  Depends heavily on the content you provide, but
+## the default is set to a week, which makes Google Chrome PageSpeed a
+## happy camper.
+##
+##    Cache-Control: max-age=SEC
+##
+## Min max-age value 0 (browser caching disabled)
+## Max max-age value 31536000 (1 year)
+##
+## Recommended: 604800 (a week)
+#max-age = 604800
diff -Naur orig.merecat/merecat.service.in merecat/merecat.service.in
--- orig.merecat/merecat.service.in	2019-04-22 03:53:33.036317659 -0700
+++ merecat/merecat.service.in	2019-04-22 03:53:50.002496447 -0700
@@ -4,7 +4,7 @@
 Documentation=file:@DOCDIR@/README.md
 Documentation=http://github.com/troglobit/merecat
 Documentation=http://merecat.troglobit.com
-ConditionPathExists=/var/www
+ConditionPathExists=/srv/http
 After=network-online.target
 Requires=network-online.target
 
@@ -12,7 +12,7 @@
 Type=simple
 Restart=always
 RestartSec=3
-ExecStart=@SBINDIR@/merecat -n @WWWDIR@
+ExecStart=@SBINDIR@/merecat -n
 
 [Install]
 WantedBy=multi-user.target
diff -Naur orig.merecat/merecats.service.in merecat/merecats.service.in
--- orig.merecat/merecats.service.in	1969-12-31 16:00:00.000000000 -0800
+++ merecat/merecats.service.in	2019-04-22 16:15:00.582272619 -0700
@@ -0,0 +1,18 @@
+[Unit]
+Description=Merecat web server
+Documentation=man:merecat(8) man:ssi(8) man:redirect(8) man:htpasswd(1)
+Documentation=file:@DOCDIR@/README.md
+Documentation=http://github.com/troglobit/merecat
+Documentation=http://merecat.troglobit.com
+ConditionPathExists=/srv/http
+After=network-online.target
+Requires=network-online.target
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=3
+ExecStart=@SBINDIR@/merecat -I merecats -f @SYSCONFDIR@/merecats.conf -n
+
+[Install]
+WantedBy=multi-user.target
