diff -Naur orig/app/ui.js noVNC-1.7.0-beta/app/ui.js
--- orig/app/ui.js	2026-03-26 21:33:45.160735157 -0700
+++ noVNC-1.7.0-beta/app/ui.js	2026-03-26 21:42:51.230844607 -0700
@@ -17,6 +17,7 @@
 import Keyboard from "../core/input/keyboard.js";
 import RFB from "../core/rfb.js";
 import * as WebUtil from "./webutil.js";
+import { local_settings } from "../settings.js";
 
 const PAGE_TITLE = "noVNC";
 
@@ -77,12 +78,8 @@
         // Translate the DOM
         l10n.translateDOM();
 
-        // We rely on modern APIs which might not be available in an
-        // insecure context
-        if (!window.isSecureContext) {
-            // FIXME: This gets hidden when connecting
-            UI.showStatus(_("Running without HTTPS is not recommended, crashes or other issues are likely."), 'error');
-        }
+        // In our installation this is usually sitting behind a proxy or VPN
+        // secure check removed by packager
 
         // Try to fetch version number
         try {
@@ -133,7 +130,8 @@
 
         document.documentElement.classList.remove("noVNC_loading");
 
-        let autoconnect = UI.getSetting('autoconnect');
+        let autoconnect = local_settings.autoconnect;
+
         if (autoconnect === 'true' || autoconnect == '1') {
             autoconnect = true;
             UI.connect();
@@ -172,23 +170,24 @@
         UI.setupSettingLabels();
 
         /* Populate the controls if defaults are provided in the URL */
-        UI.initSetting('host', '');
-        UI.initSetting('port', 0);
-        UI.initSetting('encrypt', (window.location.protocol === "https:"));
-        UI.initSetting('password');
-        UI.initSetting('autoconnect', false);
-        UI.initSetting('view_clip', false);
-        UI.initSetting('resize', 'off');
-        UI.initSetting('quality', 6);
-        UI.initSetting('compression', 2);
-        UI.initSetting('shared', true);
-        UI.initSetting('bell', 'on');
-        UI.initSetting('view_only', false);
-        UI.initSetting('show_dot', false);
-        UI.initSetting('path', 'websockify');
-        UI.initSetting('repeaterID', '');
-        UI.initSetting('reconnect', false);
-        UI.initSetting('reconnect_delay', 5000);
+        UI.initSetting('host',            local_settings.host);
+        UI.initSetting('port',            local_settings.port);
+        UI.initSetting('encrypt',         local_settings.encrypt);
+        UI.initSetting('view_clip',       local_settings.view_clip);
+        UI.initSetting('resize',          local_settings.resize);
+        UI.initSetting('quality',         local_settings.quality);
+        UI.initSetting('compression',     local_settings.compression);
+        UI.initSetting('shared',          local_settings.shared);
+        UI.initSetting('view_only',       local_settings.view_only);
+        UI.initSetting('show_dot',        local_settings.show_dot);
+        UI.initSetting('path',            local_settings.path);
+        UI.initSetting('repeaterID',      local_settings.repeaterID);
+        UI.initSetting('reconnect',       local_settings.reconnect);
+        UI.initSetting('reconnect_delay', local_settings.reconnect_delay);
+        UI.initSetting('password',        local_settings.password);
+        UI.initSetting('autoconnect',     local_settings.autoconnect);
+        UI.initSetting('bell',            local_settings.bell);
+
     },
     // Adds a link to the label elements on the corresponding input elements
     setupSettingLabels() {
@@ -755,10 +754,7 @@
             defVal = UI.customSettings.defaults[name];
         }
         // Check Query string followed by cookie
-        let val = WebUtil.getConfigVar(name);
-        if (val === null) {
-            val = WebUtil.readSetting(name, defVal);
-        }
+        let val = WebUtil.readSetting(name, defVal);
         WebUtil.setSetting(name, val);
         UI.updateSetting(name);
         // Has the user forced a value?
