diff -Naur orig/configure.ac fixed/configure.ac
--- orig/configure.ac	2025-03-19 00:04:59.810907325 -0700
+++ fixed/configure.ac	2025-03-19 01:33:49.215322176 -0700
@@ -134,6 +134,22 @@
 AC_MSG_NOTICE($ar_distro_id)
 DISTRO_NAME="|                         Distribution Name ... Unknown   |"
 case "$ar_distro_id" in
+     arch)
+       AC_MSG_NOTICE([Distro is Arch Linux])
+       DISTRO_FAMILY="|                      Distribution Family ... Arch Linux |"
+       DISTRO_NAME="|                        Distribution Name ... Arch Linux |"
+       AC_SUBST(APACHE_PKG,"apache")
+       AC_SUBST(APACHE_CONFIG_DIR,"/etc/httpd/conf")
+       AC_SUBST(USERMODE_PKG,"")
+       AC_SUBST(MYSQL_PKG,"mariadb-clients")
+       AC_SUBST(QT_MYSQL_PKG,"qt5-base")
+       AC_SUBST(DOC_PATH,"/usr/share/doc/rivendell")
+       AC_SUBST(HAVE_USERMODE,"no")
+       AC_SUBST(RDALSACONFIG_DESKTOP_FILE,"rivendell-rdalsaconfig-sudo.desktop")
+       AC_SUBST(RDDBCONFIG_DESKTOP_FILE,"rivendell-rddbconfig-sudo.desktop")
+	   AC_SUBST(PYPAD_DEPS, "python python-pycurl python-requests python-pyserial python-pymysql python-urllib3")
+       AC_SUBST(RIVENDELL_PAM_FILE,"rivendell-rhel.pam")  
+     ;;
      debian)
        AC_MSG_NOTICE([Distro is Debian-ish])
        DISTRO_FAMILY="|                       Distribution Family ... Debian    |"
diff -Naur orig/get_distro.pl fixed/get_distro.pl
--- orig/get_distro.pl	2025-03-19 00:04:59.687767952 -0700
+++ fixed/get_distro.pl	2025-03-19 01:02:28.119528162 -0700
@@ -26,6 +26,7 @@
 #
 
 my $usage="USAGE: get_distro.pl NAME|PRETTY_NAME|ID|ID_LIKE|VERSION|MAJOR|MINOR|POINT";
+my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) = localtime();
 
 if($ARGV[0] eq "NAME") {
     print &Extract("NAME");
@@ -43,41 +44,28 @@
 }
 
 if($ARGV[0] eq "ID_LIKE") {
-    print &Extract("ID_LIKE");
+    print &Extract("ID");
     exit 0;
 }
 
 if($ARGV[0] eq "VERSION") {
-    print &Extract("VERSION_ID");
+    printf("%d.%02d.%02d-rolling", $year + 1900, $mon + 1, $mday);
     exit 0;
 }
 
 if($ARGV[0] eq "MAJOR") {
     my $ver=&Extract("VERSION_ID");
-    my @f0=split '\.',$ver;
-    print $f0[0];
+    print $year+1900;
     exit 0;
 }
 
 if($ARGV[0] eq "MINOR") {
-    my $ver=&Extract("VERSION_ID");
-    my @f0=split '\.',$ver;
-    if(scalar(@f0)>=2) {
-	print $f0[1];
-	exit 0;
-    }
-    print "0";
-    exit 0;
+    printf("%02d", $mon+1);
+    exit;
 }
 
 if($ARGV[0] eq "POINT") {
-    my $ver=&Extract("VERSION_ID");
-    my @f0=split '\.',$ver;
-    if(scalar(@f0)>=3) {
-	print $f0[2];
-	exit 0;
-    }
-    print "0";
+    printf("%02d", $mday);
     exit 0;
 }
 

