diff -Naur signal-cli-rest-api/src/main.go new/src/main.go
--- signal-cli-rest-api/src/main.go	2026-04-15 16:41:58.840250415 -0700
+++ new/src/main.go	2026-04-15 17:48:12.931589187 -0700
@@ -79,6 +79,12 @@
 		}
 	}
 
+	log.Debug("signal rest api config dir: ", configdir);
+	log.Debug("signal attachment dir: ", attachdir);
+	log.Debug("signal avatar dir: ", avatardir);
+
+	
+
 	if utils.GetEnv("SWAGGER_USE_HTTPS_AS_PREFERRED_SCHEME", "false") == "false" {
 		docs.SwaggerInfo.Schemes = []string{"http", "https"}
 	} else {
@@ -464,5 +470,5 @@
 		c.Start()
 	}
 
-	router.Run()
+	router.Run(swaggerHost)
 }
