diff -Naur signal-cli-rest-api/src/main.go new/src/main.go
--- signal-cli-rest-api/src/main.go	2026-04-15 16:41:58.840250415 -0700
+++ new/src/main.go	2026-05-06 18:31:32.072264707 -0700
@@ -101,6 +101,7 @@
 	swaggerIp := utils.GetEnv("SWAGGER_IP", defaultSwaggerIp)
 	swaggerHost := utils.GetEnv("SWAGGER_HOST", swaggerIp+":"+port)
 	docs.SwaggerInfo.Host = swaggerHost
+	log.Debug("host to bind to: ", swaggerHost)
 
 	log.Info("Started Signal Messenger REST API")
 
@@ -464,5 +465,5 @@
 		c.Start()
 	}
 
-	router.Run()
+	router.Run(swaggerHost)
 }
